src/devices/sound/c140.cpp

```diff
@@ -463,10 +463,23 @@ void c219_device::sound_stream_update(sound_stream &stream, std::vector<read_str
 	}
 }
 
+inline u8 c140_device::keyon_status_read(u16 offset)
+{
+	m_stream->update();
+	C140_VOICE *v = &m_voi[offset >> 4];
```

Comment:
You know C++ has references.

Reply:
In what sense?

Reply:
It’s a bit prettier if you do like:
```
    C140_VOICE const &v = m_voi[offset >> 4];
...
    return (v.key ? 0x40 : 0x00) | (v.mode & 0x3f);
```

I know here it makes very little difference, but using references rather than pointers protects you from accidentally indexing it or altering the pointer when you don’t want to.

Reply:
Was just following device style, but okay I see your point.